src/components/Quiz.js

```diff
@@ -34,42 +35,42 @@ const Quiz = () => {
   return (
     <Container>
       <h1>CYF Quiz</h1>
-      <Form onSubmit={checkUserAnswers}>
-        <Question
-          key={question.id}
-          question={question}
-          handleAnswerSelect={handleAnswerSelect}
-          selectedAnswer={userAnswers[question.id]}
+      {shouldDisplayScore ?
+        <Result
+          score={score}
+          NumOfQuestions={data.length}
```

Comment:
Variable names in JS start that are not classes or React components start with a small letter

Reply:
I found it set on default, I switched it to formate with prettier. 
Fixed it. :)